### PR TITLE
Service test changes for empty file

### PR DESCRIPTION
**Pull request by bkmoovio**
Service test changes for empty file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ sudo: false
 os:
 - linux
 - osx
-- windows
+#- windows
```
Comment (adamdecaf) [inline .travis.yml:6]: Why was this commented out @bkmoovio ?

**Comment by brookekline** [comment]
Because it wouldn’t build and Wade thought there was a problem with Travis ci

Regards,

Brooke

**Comment by bkmoovio** [comment]
I think you will need to re-base to get the service_test changes to resolve the following on the empty-file-error branch.  There may be additional errors that are still there, that haven't been seen yet.

--- FAIL: TestGetFileContents (0.00s)
   service_test.go:129: problem writing plaintext file 12345: EntryAddendaCount 00000000 is a mandatory field and has a default value
--- FAIL: TestValidateFile (0.00s)
   service_test.go:149: BatchCount 000000 is a mandatory field and has a default value

**Comment by bkmoovio** [comment]
yes, the same is returned with a rerun of the empty-file-error branch

--- FAIL: TestGetFileContents (0.00s)
    service_test.go:129: problem writing plaintext file 12345: EntryAddendaCount 00000000 is a mandatory field and has a default value, did you use NewFileControl()?
--- FAIL: TestValidateFile (0.00s)
    service_test.go:149: BatchCount 000000 is a mandatory field and has a default value, did you use NewFileControl()?
FAIL

The command "go test ./..." exited with 1.
0.12s$ misspell -error -locale US $GOFILES

**Pull request merged**